src/content/docs/dev/amplifier/gmp-example.mdx

```diff
@@ -6,127 +6,141 @@ This example shows the execution of a GMP call from the Ethereum Sepolia to the
 
 You will need:
 
-- An [`axelard` node](/dev/amplifier/chain-integration/introduction/) that is [connected to the devnet](/validator/amplifier/verifier-onboarding/)
+- A [funded wallet](https://docs.axelar.dev/dev/amplifier/chain-integration/integrate-a-chain/#make-sure-you-have-a-funded-wallet) on `devnet-amplifier`
 - A [Solidity contract](https://github.com/cjcobb23/axelar-examples/blob/main/examples/evm/call-contract/CallContract.sol) deployed to the Ethereum Sepolia and Avalanche Fuji testnets
+  - We use a [sample contract](https://gist.github.com/StephenFluin/9379a9fc202d54854fd74f42bcac3178) throughout this demo
 
 ## Helpful references
 
-- [Amplifier devnet contract addresses](https://github.com/axelarnetwork/axelar-contract-deployments/pull/187/files)
+- [Amplifier devnet contract addresses](https://github.com/axelarnetwork/axelar-contract-deployments/blob/main/axelar-chains-config/info/devnet-amplifier.json)
 - [`axelar-amplifier` GitHub repository](https://github.com/axelarnetwork/axelar-amplifier)
 
 ## Send a GMP call between two chains
 
 ### Execute the contract call on the source chain
 
-Use `callContract()` to execute the contract call on Ethereum Sepolia, then locate the `ContractCall` event in the transaction details on [Axelarscan](https://testnet.axelarscan.io/), which should look something like [this](https://sepolia.etherscan.io/tx/0xeafbc1283699c49ba4f79ec43d8749fd82ca4ee14c236787dd25a2f7d4932daa). Note the transaction hash and log index.
+Use `callContract()` to execute the contract call on Ethereum Sepolia, then locate the `ContractCall` event in the transaction details on [Axelarscan](https://testnet.axelarscan.io/), which should look something like [this](https://sepolia.etherscan.io/tx/0xe279af3b1faa222d274c3f6c30bee5f86c5faf0199d277a3aa8c36071219af4c). Note the transaction hash and log index.
```

Comment:
Can we link to the axelarscan link instead? Since we're referencing axelarscan here

Reply:
Good point, also updated the previous link to point to https://devnet-amplifier.axelarscan.io